crates/python/src/lib.rs

```diff
@@ -58,6 +58,7 @@ fn qcs_sdk(py: Python<'_>, m: &PyModule) -> PyResult<()> {
         Err(e) => eprintln!("Failed to initialize the qcs_sdk logger: {e}"),
     }
     init_submodule("qcs_sdk", py, m)?;
+    pyo3_tracing_subscriber::add_submodule("qcs_sdk.tracing_subscriber", py, m)?;
```

Comment:
I'm not noticing any problems with imports as written, but just in case something unexpected comes up with them, we may need to update `sys.modules` and the module's `__name__` manually. Similar to what [rigetti-pyo3 does](https://github.com/rigetti/rigetti-pyo3/pull/35/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R234).

Reply:
Actually, `pyo3_tracing_subscriber::add_submodule` copies what's done in rigetti-pyo3 so we should be good there.